pages/posts/roadmap-2024.md

```diff
@@ -0,0 +1,219 @@
+
+
+- **Objective 2: Ship a viable solution for decentralized access to Beacon block headers**
+    -  Key result 1:
+        -  Portal Beacon subprotocol is fully featured according to the canonical specification and there is feature parity across all three Portal clients. 
+    -  Key result 2:
+        -  Beacon network never drops below 99% score on Glados
+    -  Key result 3:
+        - \>=1 application or light client using Portal Network to follow the head of the Beacon chain
+    - Key result 4:
+        - \>=1 execution client using Portal Beacon network to accelerate their syncing
+
+
+- **Objective 3: Ship a viable solution for decentralized access to state data**:
+    - Key result 1:
+        -  Portal State subprotocol is fully featured according to the canonical specification and there is feature parity across all three Portal clients. 
+    -  Key result 2:
+        -  State network never drops below 99% score on Glados
+    -  Key result 3:
+        - \>=1 application or client using Portal State network to request `eth_getProof` data
+
+- **Objective 4: Develop and maintain reliable tooling**
+    - Key result 1: 
+        - Glados maintains 99% uptime and is able to handle high volume of traffic (up to 10^5 visits/day)
+    - Key result 2:
+        - Portal Hive has 100% test coverage for all core functionality for all three networks across all three flagship Portal clients
+
+
+- **Objective 5: Communicate effectively with users and developers**
+    - Key result 1: 
+        - Github issues and comments across the Portal specs, Glados, and client repositories are always responded to (even if just an acknowledgement of receipt) in less than 3 days.
+    - Key result 2: 
+        - New features and milestones are reported via the Portal blog within 1 month of final PR meging merged.
+    - Key result 3:
+        - At least 5 presentations focused on portal Network are delivered at major Ethereum events in 2024, including Devcon.
+
+
+### 2025 and beyond
+
+These are longer term objectives that might be researched and discussed during 2024 but are not expected to be shipped until at least 2025. These will be revisited during 2025 roadmap planning.
+
+- **Objective 1: Portal Network is widely accepted as the default source for light client execution and consensus data**
+    - \> 10 apps/light clients have replaced third party centralized RPC providers with Portal Network.
+
+- **Objective 2: Portal Network is widely considered critical Ethereum infrastructure** 
+    - Key result 1: Portal mode is available is most Ethereum execution clients, either enablign them to aggressively prune their local data or to accelerate their sync times.
+
+- **Objective 3: Portal Network is supporting major new developments for Ethereum**
+
+    - Key result 1: Portal Network serves data enabling e.g. stateless clients, SNARKified EVM, or other substantial Ethereum upgrades from the protocol roadmap. 
```

Comment:
I think you could add support for verkle trees here

Reply:
```suggestion
    - Key result 1: Portal Network serves data enabling e.g. stateless clients, SNARKified EVM, Verkle trees, or other substantial Ethereum upgrades from the protocol roadmap. 
```